todoman/ui.py

```diff
@@ -343,12 +361,24 @@ def compact(self, todo):
             list=todo.list.name,
             percent=todo.percent_complete,
             summary=todo.summary,
-            # XXX: Move this into Todo itself and dedupe it
-            urgent=todo.priority not in [None, 0],
+            priority=todo.priority,
         )
 
         return json.dumps(data, sort_keys=True)
 
+    def parse_priority(self, priority):
```

Comment:
I just defined a `parse_priority` for `PorcelainFormatter`, why doesn't that work?